codes/rust/chapter_computational_complexity/space_complexity.rs

```diff
@@ -0,0 +1,73 @@
+#![allow(unused_variables)]
+/**
+ * File: space_complexity.cpp
+ * Created Time: 2023-01-13
+ * Author: xBLACICEx (xBLACKICEx@outlook.com )
+*/
+
+use std::collections::HashMap;
+
+/* 函数 */
+fn func() {
+    // do something
+}
+
+/* 常数阶 */
+fn constant(n: i32) {
+    // TODO
```

Comment:
It is required to solve all TODOs before merging.

Reply:
Then I'll remove space_complexity.rs temporary, I won't finish it in short time.

Is Ok like this?

Reply:
That's alright. You can create a new PR when finishing it.